modules/Innovation/GameState.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Innovation;
+
+class GameState
+{
+    /** @var \Table $game */
+    private $game;
+
+    public function __construct($game)
+    {
+        $this->game = $game;
+    }
+
+    /**
+     * @param string $key
+     * @return mixed
+     */
+    public function get(string $key)
+    {
+        return $this->game->getGameStateValue($key);
+    }
+
+    /**
+     * @param string $key
+     * @param $value
+     * @return mixed
+     */
+    public function set(string $key, $value)
+    {
+        return $this->game->setGameStateValue($key, $value);
+    }
+
+    /**
+     * Increment a value for a key
+     *
+     * @param string $key
+     * @param int $value
+     * @return mixed
+     */
+    public function inc(string $key, int $value = 1)
+    {
+        return $this->game->incGameStateValue($key, $value);
+    }
+
+    /**
+     * @param string $key
+     * @param $value
+     * @return void
+     */
+    public function setInitial(string $key, $value)
+    {
```

Comment:
Why do you sometimes put the brace on the next line and other times you put it on the same line? For consistency, let's put it on the same line.

Reply:
A miss on my part, need to enable auto-formatting in my IDE for this project. Fixed!

Reply:
Do you feel strongly about putting braces on the next line instead of the same line? It goes against the style we've been using thus far.

Reply:
Yep, I do - it follows PSR standards and is what the PHP community at-large follows. Not doing so is actually out of step with broader PHP standards: 
- https://www.php-fig.org/psr/psr-2/#1-overview
- https://www.php-fig.org/psr/psr-12/#44-methods-and-functions